Editor/UI/IssueTable.cs

```diff
@@ -368,6 +384,16 @@ void CellGUI(Rect cellRect, TreeViewItem treeViewItem, int columnIndex, ref RowG
             if (rule != null && rule.severity == Rule.Severity.None) GUI.enabled = true;
         }
 
+        void CenterRectUsingSingleLineHeight(ref Rect rect)
```

Comment:
`CenterRectUsingSingleLineHeight` is defined in `IMGUI.Controls.TreeView` as non virtual. If overriding this method only when accessed through references of type `IssueTable` is the intent, use the `new` keyword to get rid of the warning. This does not make the method any more `virtual` however and is a bit of a code smell to me. Since this method is only called directly above in `CellGUI()`, I suggest renaming it to avoid confusion.

Reply:
Good catch!
Since the new `CenterRectUsingSingleLineHeight` is almost identical to the base class one, I fixed it by adding the `new` keyword. For the reader's awareness I think I prefer this way.